packages/entando-apimanager/modules/api/apiManager.js

```diff
@@ -142,6 +118,94 @@ const normalizeErrorMessage = (message) => {
   return 'serverError';
 };
 
+export const authIntercept = () => {
+  if ('currentUser' in store.getState() && getRefreshToken()) {
+    authorize(getRefreshToken(), getAuthenticationToken());
+  }
+};
+
+const beginInterceptSetup = () => {
+  const configIntercept = {
+    // specify if request should be intercepted here we intercept
+    // intercepts only if refreshTokenConfig object props is specified
+    shouldIntercept: () => refreshTokenConfig !== null,
```

Comment:
why is this a function and not just a boolean?

Reply:
this is always called before an attempt to intercept fetch calls. 

It is originally `shouldIntercept: (request) => { ... }` but I don't need the request param. just the indication if `refreshTokenConfig` is provided